### ISSUE TITLE
Can only delete one cookie at a time

### ISSUE THREAD

**Issue by bskendig** [ysard/cookie-quick-manager#9]
Pressing the Delete key only deletes a single cookie at a time. If a site has six cookies, then I have to delete all six of them individually.

I'd like to be able to select a site then press Delete to get rid of all that site's cookies at once. I'd also like to be able to select multiple sites (using Shift or Ctrl to extend my selection) then press Delete once to get rid of all the cookies of all of the selected sites.

**Comment by bskendig** [comment]
I just noticed there's a trashcan icon that'll delete all of a site's cookies. Still, that's a lot of mousing back and forth to delete each site individually. It would be a lot more convenient to use the Delete key, and to be able to select multiple sites to delete.

**Comment by inliquid** [comment]
This "trash icon" seems to delete ALL cookies at a browser not just for particular search! This is very annoying issue!

**Comment by ysard** [comment]
The multiselection feature is in great demand but is a bit complex to implement for now, but I'm thinking about it.

@inliquid 
The delete button we were talking about was removed in favor of the context menu. Only the button deleting all cookies from the context (and not from the browser as the text says in the current public version) has remained. I've added a warning window to warn the user that cookies in the current context will be deleted if it continues. It's more secure like that.

**Comment by inliquid** [comment]
@ysard no, I if tell you something, that means I had experience. I wouldn't spend time "imagining" of something. So once again. By clicking on delete all this software DELETED ALL COOKIES in web browser despite there was a search request. AND WITHOUT ANY WARNINGS!

**Comment by ysard** [comment]
I never said that you imagined something. When bskendig talks about a "trashcan icon that'll delete all of a site's cookies", he's talking about a button that has been moved to the context menu. The only button remaining on the bottom left is the one you clicked. There is nevertheless an alert in the public version on addons.mozilla.org: a tooltip on mouse hover which indicates that this button "deletes all cookies" (while it only erases cookies of the current context (the default one by default )).

As I said, I added after your good remark an alert window that asks for confirmation for this dangerous action. I misspoke myself because this change is not yet published on AMO, but it's a matter of hours.
